src/applications/edu-benefits/0994/components/TargetCheckboxWidget.jsx

```diff
@@ -0,0 +1,49 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+
+export function TargetCheckboxWidget({
```

Comment:
This like it's mostly a copy of the existing checkbox widget. Is there a reason we can't use that one directly?